src/lib/FullCalendar.svelte

```diff
@@ -124,11 +124,10 @@
 	<article>
 		<h2>Add to Your Calendar</h2>
 		<p>
-			Copy the iCal url into your calendar app to automatically add Kendo Club events to your
-			calendar.
+			Copy this iCal link into your calendar app to subscribe to the Kendo Club at Umich calendar.
```

Comment:
Did you change URL to link intentionally?

Reply:
No I don't think so

Reply:
Would you like to change it back?

Reply:
Sorry, wasn't able to at the time. Just did so.